src/models/launch_2/repairs.md

```diff
@@ -0,0 +1,158 @@
+
+### Steps to remove the PCB:
+
+1. [Remove all of the keycaps](#removing-and-installing-keycaps) and [all of the switches](#removing-and-installing-switches).
+    - The 2U stabilizers do not need to be removed.
+
+![Launch with all keycaps removed](./img/all-keycaps-removed.webp)
+![Launch with all switches removed](./img/all-switches-removed.webp)
+
+2. Turn the keyboard over and remove the five bottom panel screws.
+    - The screws are set into adapters that may or may not come out with the screws.
+
+![Bottom panel screws](./img/bottom-panel-screws.webp)
+
+3. Remove the bottom panel.
+    - If the magnetic lift bar is installed, it can be used as a handle.
+
+![Bottom panel removed](./img/internal-bottom.webp)
+
+4. Push the PCB up out of the chassis through one of the switch holes, then lift the PCB out of the chassis.
+    - To avoid the USB-C port catching on the chassis, push through one of the bottom row switch holes and lift the bottom edge of the PCB first.
+
+![Loose PCB](./img/pcb-removed.webp)
+
+### Steps to install the PCB:
+
+1. Install all of the switches into the chassis.
+    - The switches can also be installed after installing the PCB, but installing the switches first is recommended because it reduces the risk of individual bent pins damaging the PCB or switches during installation.
+    - Make sure the switches are in the correct orientation with the LED window at the top.
+    - The switches are not specific to their key/hole.
+
+![Switches installed in chassis](./img/chassis-with-switches.webp)
+
+2. Turn the chassis over and check that all of the switch pins are straight.
+    - Look down the rows of pins from the sides of the chassis to identify any bent pins.
+    - If any of the pins are bent when installing the PCB, the switch or the PCB may be damaged.
+    - In the image below, the pin marked with a red arrow is an example of a pin that is bent and needs to be straightened.
+
+![Switch pins in chassis (with a bent pin)](./img/chassis-switch-pin-check.webp)
+
+3. Carefully line up the PCB with the switch pins and place it in the chassis, starting with the USB-C ports on the top edge.
+    - Evenly press the PCB down until all of the switch pins are fully seated in their sockets.
+    - It may help to pick up the keyboard and pinch to press both the PCB and the switches at the same time.
+    - If you feel excessive resistance, stop, remove the PCB, and ensure that all of the switch pins are lined up.
+    - The switch pins can be seen from the back of the sockets when they are fully seated, highlighted below.
+
+![Switch pins visible through sockets](./img/pcb-switch-pins.webp)
+
+4. Install the bottom cover and its four screws, flip the keyboard over, and put all of the keycaps back on.
+    - If any of the oval steel inserts came loose from the bottom cover, put them back into place before reinstalling the bottom cover.
```

Comment:
@leviport What's the reasoning behind calling these "steel inserts" instead of "magnets?" Is there a technical difference I'm unaware of? (The current Lite section just calls them magnets.)

Reply:
They aren't magnets, they're just pieces of steel for the magnets in the angle bars to stick to. I guess I missed it in that section of Lite.

Reply:
Ok, I guess ferromagnetic metals shouldn't necessarily be called magnets. Thanks.